src/interfaces/jira.ts

```diff
@@ -83,7 +83,7 @@ export interface JiraDeployment {
 	displayName: string;
 	url: string;
 	description: string;
-	lastUpdated: number;
+	lastUpdated: string;
```

Comment:
I think this should actually be a `Date` since Date will stringify to UTC string - `JSON.stringify({date:new Date()})` becomes `'{"date":"2021-12-02T03:20:46.554Z"}'`